Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -2314,6 +2316,14 @@ def condition(self, fragment, stepList, key, hasHarvest):
         'BeamSpot': 'Realistic25ns13p6TeVEarly2022Collision',
         'ScenToRun' : ['GenSim','Digi','RecoNano','HARVESTNano','ALCA'],
     },
+    '2023FS' : {
+        'Geom' : 'DB:Extended',
+        'GT' : 'auto:phase1_2023_realistic',
+        'HLTmenu': '@relval2022',
```

Comment:
This should eventually change to `HLT:@relval2023` in light of #41017 (and https://github.com/cms-sw/cmssw/pull/41007). If this PR is integrated before those, #41017 will need to be rebased to change this to `@relval2023`.

Reply:
Thanks @missirol 
I will let #41017 and #41007 merge first. This PR can be rebases, it is very small.

Reply:
Thanks, @srimanob ! Could you please also have a look at #41016 (needs upgrade signature) ?